xla/service/gpu/tests/gpu_fused_mha_test.cc

```diff
@@ -1102,6 +1102,177 @@ class FlashAttentionBMMScalePaddingMaskSoftmaxBMM
+      get-tuple-element.14 = u8[0]{0} get-tuple-element(custom-call.10), index=3
+      get-tuple-element.11 = bf16[4,4,1024,64]{3,1,2,0} get-tuple-element(custom-call.10), index=0
+      transpose.15 = bf16[4,1024,4,64]{3,2,1,0} transpose(get-tuple-element.11), dimensions={0,2,1,3}
+      get-tuple-element.12 = bf16[4,4,1024,64]{3,1,2,0} get-tuple-element(custom-call.10), index=1
+      transpose.16 = bf16[4,1024,4,64]{3,2,1,0} transpose(get-tuple-element.12), dimensions={0,2,1,3}
+      get-tuple-element.13 = bf16[4,4,1024,64]{3,1,2,0} get-tuple-element(custom-call.10), index=2
+      transpose.17 = bf16[4,1024,4,64]{3,2,1,0} transpose(get-tuple-element.13), dimensions={0,2,1,3}
+      ROOT tuple.18 = (bf16[4,1024,4,64]{3,2,1,0}, bf16[4,1024,4,64]{3,2,1,0}, bf16[4,1024,4,64]{3,2,1,0}, bf16[4,1024,4,64]{3,2,1,0}) tuple(transpose.9, transpose.15, transpose.16, transpose.17)
+    } // main.19
+    )";
+    return hlo_text;
+  }
+
+  template <typename T>
+  void TestImpl_Flash_Attention_Training_BMM1_SlidingWindowMask_Softmax_BMM2() {
+    if (skip_reason_) GTEST_SKIP() << *skip_reason_;
+    if (GetDnnVersionInfoOrDefault(backend().default_stream_executor()) <
+        se::dnn::VersionInfo(9, 0, 0)) {
+      GTEST_SKIP() << "Flash Attention requires cuDNN >= 9.0.0.";
```

Comment:
It seems this needs to check for cudnn 9.2.0. We get this error message internally:

For cuDNN version below 9.2.0, sliding window attention is not supported